Building Portfolio Set Up and Mobile first project

lucy-sees opened the pull request:
Please review this work for merge. Here are the features of my project:

- Created a branch called background-edit.
- Used Flex box in my HTML code as per the requirements.
- Used all CSS properties on my stylesheet as displayed in figma.
- Implemented some links and contact buttons

mukhammadrizooff reviewed (commented):
Hi @lucy-sees, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that ### Hi @lucy-sees, Please add descriptive information about your changes and project to your `Pull Request` summary and chnage your current `README` information to latest version [README template](https://github.com/microverseinc/readme-template) 🙏🏼 
#
![image](https://user-images.githubusercontent.com/63915024/205689452-44acf119-9bb6-4299-a14f-0a9aba919658.png)
#
![image](https://user-images.githubusercontent.com/63915024/205690803-4e7963c0-1396-40a5-b8d5-48b9b7472979.png)
#


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

lucy-sees commented:
Please review this work for merge, I have changed the previous README information to match the latest version [README template](https://github.com/microverseinc/readme-template) 🙏🏼

ha-manel reviewed (commented):
# STATUS: INVALID 🔴 
Hi @lucy-sees 
I'm sure you put a lot of effort into this project but this PR is invalid because you don't have a descriptive pull request title and description😓as pointed out and explained by the previous reviewer, kindly edit your pull request's title and description to make them descriptive of the changes that you made in this project 👍 

Also, your `linters.yml` is empty and linter checks are not running, kindly follow this [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) to setup linters correctly 💯 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lucy-sees commented:
I have made the following corrections to make my project PR valid 
- managed to set up linters correctly and now all checks are running
- updated my pull request title and made it more descriptive
- updated the pull request description and made it more descriptive

roshan-bajgain commented on index.html:
```diff
@@ -34,7 +35,7 @@ <h2>LET'S CONNECT</h2>
               <i> <img id="linkedin" src="images/linkedin.png"></i>
               <i><img id="m" src="images/m.png"></i>
-              <i><img id="cat" src="images/cat.png"></i>
+              <i><img id="github" src="images/github.png"></i>
               <i><img id="peace" src="images/peace.png"></i>
           </ul>     
       </div>
```
- [ ] I would suggest you please replace the `div` tag with a section tag because It represents a generic standalone section of the document.

lucy-sees replied:
I have managed to replace the div tag with a section tag

roshan-bajgain commented on index.html:
```diff
@@ -34,7 +35,7 @@ <h2>LET'S CONNECT</h2>
               <i><img id="twitter" src="images/twitter.png"></i> 
               <i> <img id="linkedin" src="images/linkedin.png"></i>
               <i><img id="m" src="images/m.png"></i>
-              <i><img id="cat" src="images/cat.png"></i>
+              <i><img id="github" src="images/github.png"></i>
               <i><img id="peace" src="images/peace.png"></i>
```
- [ ] Please wrap this social icon with an `a` tag for a better semantic reason

lucy-sees replied:
I have also wrapped the icons with anchor tags for better semantics

lucy-sees commented:
> I have made the following corrections to make my project PR valid
> 
> * managed to set up linters correctly and now all checks are running
> * updated my pull request title and made it more descriptive
> * updated the pull request description and made it more descriptive

@mukhammadrizooff

roshan-bajgain commented on styles.css:
```diff
@@ -1,153 +1,153 @@
+  flex-grow: 0;
+}
+
+.text h2 {
+  width: 120px;
+  height: 24px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```
- [ ] You should stick to the design as much as possible like color, font, background, and spacing. Also, the gap between the social icons Please adjust the spacing. See the image below for reference.
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/206854637-ff3c6e48-198e-4be5-83b1-afd559307232.png)

### Your design
![Screenshot 2022-12-10 175835](https://user-images.githubusercontent.com/81727988/206854644-ae53bc50-f50b-483a-a349-d6ed953d4d63.png)

roshan-bajgain commented on styles.css:
```diff
@@ -1,153 +1,153 @@
+  background-attachment: scroll;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```
- [ ] [Optional] You should stick to the design as much as possible. Please use `br` tag so It matches the design
### Your design
![s](https://user-images.githubusercontent.com/81727988/206854803-30a688b1-3607-4cb4-9439-c5878da763bc.png)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/206854800-34001100-53e3-4c6c-bdbe-632ead8ea308.png)

roshan-bajgain reviewed (changes requested):
Hi @lucy-sees, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Please stick to the design as much as possible. please fix the spacing that is on the right and also remove the black border. See the image below for reference
- [ ] There is more gap between the heading and the `h1` tag please decrease it.
### Y0ur design
![Screenshot 2022-12-10 180634](https://user-images.githubusercontent.com/81727988/206854917-0930ef04-7b86-4eb2-aa05-002bf8707f79.png)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lucy-sees commented:
here are some of the changes I've made

- used br tag so It matches the design
- Adjusted the spacing between the social icons

roshan-bajgain reviewed (approved):
Hi @lucy-sees, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
